calculus/integration-partial-fractions.md

```diff
@@ -1 +1,134 @@
 # Partial fraction decomposition
+
+
+Some ODEs give rise to integrals like:
+
+
+\begin{equation}
+y' = \frac{3x+11}{x^2-x-6} \to y = \int \frac{3x+11}{x^2-x-6} \, dx
+\end{equation}
+
+
+
+
+
+This cannot be done with u-substitution... how to take it?
+
+\begin{equation}
+\frac{3x+11}{x^2-x-6} =\frac{3x+11}{(x-3)(x+2)} = \frac{A_1}{x-3} + \frac{A_2}{x+2}
+\end{equation}
+
+ 
+
+\begin{align}
+3x+11 &= A_1(x+2) + A_2(x-3) \\
+&= (A_1 + A_2)x + (2A_1 - 3A_2) \\
+\end{align}
+
+\begin{align}
+ A_1  + A_2 = 3
+\end{align}
+
+\begin{align}
+2A_1 - 3A_2 = 11
+\end{align}
```

Comment:
```suggestion
\begin{align}
 A_1  + A_2 &= 3 \\
 2A_1 - 3A_2 &= 11
\end{align}
```

These two equations make up a single linear system of equations that gets solved in the line below, so they should be grouped together.